ledger-core/src/core/Services.cpp

```diff
@@ -133,11 +134,11 @@ namespace ledger {
             return _configuration;
         }
 
-        const std::string &Services::getTenant() const {
+        std::string Services::getTenant() const {
             return _tenant;
         }
 
-        const std::string Services::getPassword() const {
+        std::string Services::getPassword() const {
             return _password;
```

Comment:
What is the purpose of creating a copy of tenant and password strings instead of returning a reference to its ?

Reply:
Implementing the interface. Mostly. :D